bazarr/sonarr/sync/parser.py

```diff
@@ -12,7 +12,17 @@
 from .converter import SonarrFormatVideoCodec, SonarrFormatAudioCodec
 
 
-def seriesParser(show, action, tags_dict, serie_default_profile, audio_profiles):
+def get_matching_profile(tags, language_profiles):
+    matching_profile = None
+    if len(tags) > 0:
+        for _, tuple in enumerate(language_profiles):
```

Comment:
Why do you need the enumerate if we are ignoring the index? What is the shape of `language_profiles` here?